app/templates/sensor-button-toggle.html

```diff
@@ -0,0 +1,9 @@
+<span class="sensor-text-value">
+    <div ng-show="sensor.data.properties.value !== undefined">
+        <span ng-show="sensor.data.properties.value === true" class="balanced">True</span>
```

Comment:
Hi, @siovene. Thanks for your comments. I agree with the `Movement detected/No movement` and `Pressed/Released` would be better choices, and I'd like to update the description. But for the `Toggle/Not Toggle`, it seems that they can not nice to reflect the state of the button. Forward comment from [button-toggles.js][]: `It behaves differently in that pressing the button will toggle the value between 'true' and 'false' (instead of being 'true' when the button is pressed and 'false' otherwise)`. Actually, whatever the value `true` or `false`, they all reflect the state of the button, so maybe using their initial value make people easier to understand the status. How do you think about it?

[button-toggles.js]: https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/js-servers/button-toggle.js#L2